src/constants/ssm-parameter-paths.ts

```diff
@@ -58,6 +59,10 @@ export const NAMED_SSM_PARAMETER_PATHS: NamedSsmParameterPaths = {
     path: `${VPC_SSM_PARAMETER_PREFIX}/primary/subnets/public`,
     description: "A comma-separated list of public subnets",
   },
+  DevxGlobalDistBucket: {
+    path: "/account/services/devx.global.bucket",
```

Comment:
Does `global` imply the bucket is reachable outside of the org? How about `/organisation/services/artifact.bucket`?

Reply:
this feels a lot better to me :)